data/themes/default/style.css

```diff
@@ -540,28 +540,47 @@ TrackContainerView QLabel
 /* instrument pattern */
 PatternView {
 	color: rgb( 119, 199, 216 );
+	qproperty-mutedColor: rgb( 80, 80, 80 );
+	qproperty-selectedColor: rgb( 0, 125, 255 );
 	qproperty-fgColor: rgb( 187, 227, 236 );
+	qproperty-fgMutedColor: rgb( 128, 128, 128 );
 	qproperty-textColor: rgb( 255, 255, 255 );
+	qproperty-textShadowColor: rgb( 0, 0, 0 );
+	qproperty-gradient: true;
 }
 
 /* sample track pattern */
 SampleTCOView {
 	color: rgb( 74, 253, 133 );
+	qproperty-mutedColor: rgb( 80, 80, 80 );
+	qproperty-selectedColor: rgb( 0, 125, 255 );
 	qproperty-fgColor: rgb( 187, 227, 236 );
+	qproperty-fgMutedColor: rgb( 128, 128, 128 );
```

Comment:
I was thinking about this. Maybe we can avoid a lot of this by using the `background-color` property in CSS. So, `fgColor` becomes `color` and `color` becomes `background-color`.

Also we could avoid the muted and selected properties by adding custom css pseudo states like
`AutomationPatternView:muted` or `AutomationPatternView:selected`